frappe/utils/nestedset.py

```diff
@@ -261,6 +261,10 @@ def __setup__(self):
 		if self.meta.get("nsm_parent_field"):
 			self.nsm_parent_field = self.meta.nsm_parent_field
 
+	def after_insert(self):
```

Comment:
If subclass fails to call super() then this won't work btw.

Reply:
Should be fine I guess, people are expected to always call superclass hooks right?